jvm/src/main/kotlin/app/cash/security_sdk/CertificateUtil.kt

```diff
@@ -44,6 +49,33 @@ object CertificateUtil {
     )
   }
 
+  /**
+  -   * Signs CertificateRequest using the provided trifle content signer and issuing certificate.
+  -   *
+  -   * @param issuerCertificate trifle certificate associated with the signer of this cert.
+  -   * @param certificateRequest certificate request used to generate a new certificate
+  -   * @param contentSigner content signer used to generate the signature validating the certificate
+  -   */
+    fun signCertificate(
```

Comment:
I think theres some indenting issues

Reply:
ack, I made a git mistake and didn't clean it all up properly (took this from git diff)

Reply:
cmd+alt+L to autoindent